docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -175,35 +163,43 @@ This should not even be possible, but it means that an unhandled error was raise
 > Event Prefix: `Summarizer:Running:`
 
 - `summarizeCount` - the number of summarize attempts this client has made. This can be used to correlate events for individual summary attempts.
+- `summarizerSuccessfulAttempts` - the number of successful summaries this summarizer instance has performed
 
 ### SummaryAckWaitTimeout
 
 > Error
 
 When a summary op is sent, the summarizer waits `summaryAckWaitTimeout` for a summary ack/nack op in response from the server. If a corresponding response is not seen within that time, this event is raised, and the client retries.
 
-- `refSequenceNumber` - last attempt summary op reference sequence number.
+- `maxAckWaitTime` - cap on the maximum amount of time client will wait for a summarize op ack
+- `referenceSequenceNumber` - last attempt summary op reference sequence number.
 - `summarySequenceNumber` - last attempt summary op sequence number.
 - `timePending` - time spent waiting for a summary ack/nack as computed by client.
 
 ### MissingSummaryAckFoundByOps
 
 During first load, the wait for a summary ack/nack op in response to a summary op, can be bypassed by comparing the op timestamps. Normally a timer is used while running, but if the server-stamped op time difference exceeds the `maxAckWaitTimeout`, then raise this event, clear the timer and stop waiting to start.
 
-- `refSequenceNumber` - last attempt summary op reference sequence number.
+- `referenceSequenceNumber` - last attempt summary op reference sequence number.
 - `summarySequenceNumber` - last attempt summary op sequence number.
 
 ### SummarizeAttemptDelay
 
-- `retryAfterSeconds` - delay from `retryAfter` found in the summary nack response op. This will override any regular delay time.
```

Comment:
Would be useful to mention if it fires before waiting or after waiting for `delay`